test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1431,18 +1436,15 @@ public void help() throws Exception {
     }
   }
 
-  // TODO - evaluate this test is testing what is expected or history is working.
   @Test
   public void history() throws Exception {
-    final String table = name.getMethodName();
+    final String table = getUniqueNames(1)[0];
 
     ts.exec("history -c", true);
```

Comment:
There may be an opportunity here to validate that the clean command worked as expected.

Reply:
Good idea.

Reply:
I added a short validity check for the clear command in my latest commit